libs/tracing/unix/Tracing.ml

```diff
@@ -91,9 +148,27 @@ let with_setup f =
     | Some url -> url
     | None -> default_endpoint
   in
+  let data () =
+    [
+      ("version", `String config.version);
+      ("folder", `String (current_working_folder ()));
+      ("repo_name", `String (repo_name ()));
+      ("pro_secrets_validators", `Bool config.analysis_flags.secrets_validators);
+      ("pro_historical_scanning", `Bool config.analysis_flags.historical_scan);
+      ("pro_deep_intrafile", `Bool config.analysis_flags.deep_intra_file);
+      ("pro_deep_interfile", `Bool config.analysis_flags.deep_inter_file);
+    ]
+    @
+    if config.analysis_flags.secrets_validators then
+      [
+        ( "pro_secrets_allowed_origins",
+          `String (allowed_origins config.analysis_flags.allow_all_origins) );
+      ]
+    else []
+  in
   let config = Opentelemetry_client_ocurl.Config.make ~url () in
   Opentelemetry_client_ocurl.with_setup ~config () @@ fun () ->
-  with_span ~__FILE__ ~__LINE__ "All time" @@ fun _sp -> f ()
+  with_span ~__FILE__ ~__LINE__ ~data "All time" @@ fun sp -> f sp
```

Comment:
Not a huge fan of the `"All time"` name with how it shows up in Datadog under Resources. Could we just have this be `semgrep` maybe?